docs/AboutUs.md

```diff
@@ -38,15 +38,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 
-### Jean Doe
+### Rayyan Ismail
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/rayyanismail.jpg" width="200px">
 
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/rayyan35p)]
+[[portfolio](team/rayyan.md)]
```

Comment:
PPP shld be named with your github username (in lower case) :)

Reply:
Noted on the feedback, thank you!